locale/en/download/package-manager.md

```diff
@@ -289,6 +290,28 @@ Or install a binary package (if available for your platform) using pkgin:
 pkgin -y install nodejs
 ```
 
+## nvm
+On Unix / OS X systems Node.js built from source can be installed by
+[nvm](https://github.com/creationix/nvm) by installing into the location that nvm expects:
+
+```bash
+$ make install DESTDIR='$(NVM_DIR)/versions/node/' PREFIX='$(FULLVERSION)-pre'
```

Comment:
It will only be found if `nvm.sh` is sourced, and `NVM_DIR` should only be set if the profile file is sourced (which sources `nvm.sh` for you). If necessary, you'll want to source `nvm.sh` in the Makefile